flipperdevices/flipperzero-good-faps
# Picopass: Save unknown blocks with ??

# What's new

- Read/Save picopass files with "??" for unknown bytes

# Verification 

- Read an iClass SE card
- Save partial
- Open .picopass file
- See "??"

# Checklist (For Reviewer)
